app/models/turbo/streams/tag_builder.rb

```diff
@@ -77,8 +77,8 @@ def remove_all(targets)
   #   <%= turbo_stream.replace "clearance_5" do %>
   #     <div id='clearance_5'>Replace the dom target identified by clearance_5</div>
   #   <% end %>
-  def replace(target, content = nil, **rendering, &block)
-    action :replace, target, content, **rendering, &block
+  def replace(target, content = nil, method = nil, **rendering, &block)
```

Comment:
@AlexKovynev 

maybe this should be a keyword argument instead of a positional argument so one can pass rendering options without specifying the method?

btw, thanks for opening this PR :raised_hands:

Reply:
keyword with default attributes can't be mixed with ordinal. I tried.

Reply:
DONE but i am not sure that mixing is a good way. Anyway check please